### PR TITLE
Mapping With Rewards 2nd Bounties Announcement

### PR DESCRIPTION
**Pull request by pishifat**
### Description

add mapping with rewards banner and results announcement

### Status

finished, needs review

### Changelog

- added mwr banner
- added news post

### CODE REVIEW

File: news/2017-11-29-mapping-with-rewards-2-bounties-complete.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+layout: post 
+title: "Mapping With Rewards #2 - Bounties Complete!"
+date: 2017-11-29
+--- 
+
+Long overdue, but not forgotten. The second set of mapping bounties have finally been awarded and the game client has three new bundled maps!
+
+![](/wiki/shared/news/<post filename>/mwr-banner.jpg)
+
+Never thought you’d see this day, huh? Over two years ago, “Mapping With Rewards” was introduced to the public as a way to reward users who created maps for osu!’s officially licensed music -- namely tracks by [cYsmix](https://osu.ppy.sh/beatmaps/artists/2).
```
Comment (MegaApplePi) [inline news/2017-11-29-mapping-with-rewards-2-bounties-complete.md]: - `’` -> `'`
- `“` and `”` -> `"`
(and repeat for other occurrences)

Comment (pishifat) [reply news/2017-11-29-mapping-with-rewards-2-bounties-complete.md]: applied

File: news/2017-11-29-mapping-with-rewards-2-bounties-complete.md [hunk 2]
```diff
@@ -8,22 +8,22 @@ Long overdue, but not forgotten. The second set of mapping bounties have finally
 
 ![](/wiki/shared/news/<post filename>/mwr-banner.jpg)
```
Comment (Okorin) [inline news/2017-11-29-mapping-with-rewards-2-bounties-complete.md]: `wiki/shared/news/banners/mwr-banner.jpg`

you put the banner there so filename for `<post filename>` syould reflect `2017-11-29-mapping-with-rewards-2-bounties-complete` as a folder and that's where the file should be

File: news/2017-11-29-mapping-with-rewards-2-bounties-complete.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post 
+title: "Mapping With Rewards #2 - Bounties Complete!"
+date: 2017-11-29
```
Comment (Okorin) [inline news/2017-11-29-mapping-with-rewards-2-bounties-complete.md]: `date: 2017-12-01 hh:mm:ss: +0000`
time of post missing

File: news/2017-11-29-mapping-with-rewards-2-bounties-complete.md (hunk 2 above)
Comment (pishifat) [reply news/2017-11-29-mapping-with-rewards-2-bounties-complete.md]: applied

File: news/2017-11-29-mapping-with-rewards-2-bounties-complete.md (hunk 3 above)
Comment (pishifat) [reply news/2017-11-29-mapping-with-rewards-2-bounties-complete.md]: delayed

**Comment by Okorin** [comment]
oh lol sorry now you need to change my username in that post as well I guess

File: news/2018-02-02-mapping-with-rewards-2-bounties-complete.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Fright March: [Gero's rendition](https://osu.ppy.sh/s/486142)
+
+- Moonlight Sonata: [Okoratu's rendition](https://osu.ppy.sh/s/486535)
+
+- Classic Pursuit: [-kevincela-'s rendition](https://osu.ppy.sh/s/488238)
+
+Each of these winners were collectively decided by [Desperate-kun](https://osu.ppy.sh/u/2202645), [Mir](https://osu.ppy.sh/u/8688812), [-Mo-](https://osu.ppy.sh/u/2202163), [pishifat](https://osu.ppy.sh/u/3178418), and [Pentori](https://osu.ppy.sh/u/7452237). You can find more information about their judging [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vS34U2QDZTuggEIqCS4USZ2Ppq8yQk2CU_SUnI_ahJ2Dndm4S-wTD2Bhft1Say9PoRGOXDY_yCypMmO/pubhtml)!
+
+With this round finally out of the way, we'll now be able to focus all our attention on the newest round of Mapping With Rewards. There's [a bit over one month left for your submissions](https://osu.ppy.sh/community/contests/52), so be sure to give it a try if you're interested in earning some mapping bounties!
```
Comment (Xinnoh) [inline news/2018-02-02-mapping-with-rewards-2-bounties-complete.md]: Submissions for MWR 2 have closed, needs some rewording

File: news/2018-02-02-mapping-with-rewards-2-bounties-complete.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+### The Winners
+
+- Fright March: [Gero's rendition](https://osu.ppy.sh/s/486142)
+
+- Moonlight Sonata: [Okoratu's rendition](https://osu.ppy.sh/s/486535)
+
+- Classic Pursuit: [-kevincela-'s rendition](https://osu.ppy.sh/s/488238)
+
+Each of these winners were collectively decided by [Desperate-kun](https://osu.ppy.sh/u/2202645), [Mir](https://osu.ppy.sh/u/8688812), [-Mo-](https://osu.ppy.sh/u/2202163), [pishifat](https://osu.ppy.sh/u/3178418), and [Pentori](https://osu.ppy.sh/u/7452237). You can find more information about their judging [here](https://docs.google.com/spreadsheets/d/e/2PACX-1vS34U2QDZTuggEIqCS4USZ2Ppq8yQk2CU_SUnI_ahJ2Dndm4S-wTD2Bhft1Say9PoRGOXDY_yCypMmO/pubhtml)!
```
Comment (Xinnoh) [inline news/2018-02-02-mapping-with-rewards-2-bounties-complete.md]: Desperate -> Myxomatosis

File: news/2018-02-02-mapping-with-rewards-2-bounties-complete.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+Never thought you'd see this day, huh? Over two years ago, "Mapping With Rewards" was introduced to the public as a way to reward users who created maps for osu!'s officially licensed music -- namely tracks by [cYsmix](https://osu.ppy.sh/beatmaps/artists/2). 
+
+Two rounds of mapping bounties were held: one in [September 2015](https://blog.ppy.sh/post/129936251068/mapping-with-rewards-sep-2015) and another in Halloween themed one in [October 2015](https://blog.ppy.sh/post/132009865043/mapping-with-rewards-oct-2015). The first of those concluded successfully with its mappers rewarded their bounties as promised. The second, however, hadn't received as much participation and was unfortunately abandoned.
+
+...Until now, that is. A [new round of Mapping With Rewards](https://osu.ppy.sh/home/news/2017-09-23-mapping-with-rewards-returns) is currently underway, so we've brought the old Halloween round back from the dead to reward its mappers their respective bounties!
+
+### The Winners
+
+- Fright March: [Gero's rendition](https://osu.ppy.sh/s/486142)
```
Comment (Xinnoh) [inline news/2018-02-02-mapping-with-rewards-2-bounties-complete.md]: News posts are moving towards the new site, change /u/ to /users/

**Pull request merged**